File: assets/samples/domain.availability/perl/domain.avail.pl
#!/usr/bin/perl

my $domain = 'example.com';
my $apiKey = 'Your Domain Availability API Key';

my $url = 'https://domain-availability.whoisxmlapi.com/api/v1?'
        . "apiKey=$apiKey&domainName=$domain"

print get($url);

File: assets/samples/domain.availability/nodejs/domain.avail.js
var http = require('http');

var domain = 'example.com';
var apiKey = 'Your domain availability api public key';

var url = 'https://domain-availability.whoisxmlapi.com/api/v1?'
    + 'apiKey=' + apiKey + '&domainName=' + domain;

http.get(url, function(response) {
    var str = '';
    response.on('data', function(chunk) { str += chunk; });
    response.on('end', function() { console.log(str); });
}).end();
